libs/sdk-backend-tiger/src/backend/tigerSpecificFunctions.ts

```diff
@@ -882,15 +913,14 @@ export const buildTigerSpecificFunctions = (
     testDataSourceConnection: async (connectionData: IDataSourceTestConnectionRequest, id?: string) => {
         try {
             return await authApiCall(async (sdk) => {
-                const promise =
-                    id && isEmpty(connectionData)
-                        ? sdk.scanModel.testDataSource({
-                              dataSourceId: id,
-                              body: connectionData,
-                          })
-                        : sdk.scanModel.testDataSourceDefinition({
-                              testDefinitionRequest: connectionData,
-                          });
+                const promise = id
+                    ? sdk.scanModel.testDataSource({
```

Comment:
please could you explain why the isEmpty() now removed? it may now explain why nobody noticed as probably the body (or now testRequest) was empty when sent (and therefore maybe ignored)

Reply:
explained that for existing id it's now supported even without the connectionData, so no need to check